rebasehelper/checkers/abipkgdiff_tool.py

```diff
@@ -121,9 +124,33 @@ def run_check(cls, results_dir, **kwargs):
     @classmethod
     def parse_abi_logs(cls, reports):
         """
-        Parse abipkgdiff logs
-        :param reports: dictionary with paths to the logs of the produced (sub)packages
-        :return: returns dict of packages names with its abipkgdiff changes
+        Parses summary information from abipkgdiff logs.
+
+        Sets abi_changes attribute if there are any ABI changes found.
+
+        Args:
+            reports(dict): Dictionary mapping package names to abipkgdiff return codes.
+
+        Returns:
+            dict: Dictionary mapping package names to a dict of summary information for each shared object.
+
+            For example:
+
+            {
+                'libtiff':
+                {
+                   'libtiff.so.5.2.5':{
+                        'Functions changes summary': {
+                            'Added': {
+                                'count': '8',
+                                'what': 'Added',
+                                'filtered_out': None,
+                            },
+                        'Variables changes summary': {},
+                        },
```

Comment:
I have the feeling that these two lines should be swapped. The closing bracket is the end of `Functions changes summary` dict, right?